src/items/equipment/knives/knife.js

```diff
@@ -0,0 +1,19 @@
+import Equipment from "../equipment.js";
+import { EQUIPMENT } from "../../../constants/constants.js";
+
+export default class Knife extends Equipment {
+    // Attack types
+    slot = EQUIPMENT.SLOTS.WEAPON;
+    skill = EQUIPMENT.WEAPON_TYPES.RANGED;
+
+    // Attack animation
+    animation = {
+        imageName: "bronze-arrow",
```

Comment:
Should definitely be using a knife for this although may be out of scope for this

Reply:
Are there a lot of animations that need to be added/updated or just these two? If it's just these two I might do them, otherwise I'd do animations as another PR I think

Reply:
I _think_ these are the only ones that need updating now